### PR TITLE
add search option to base query its used by combined_search_by

### PR DESCRIPTION
**Pull request by adamszeptycki**

### CODE REVIEW

**Comment by adamszeptycki** [comment]
@revmischa if you could release new version if you are ok with that it would be awesome

**Comment by revmischa** [comment]
Sure, let me clean up a couple things here first

**Comment by adamszeptycki** [comment]
@revmischa ping me when it's published I want to use it in refactoring of my PR

File: jetkit/db/__init__.py [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+            or_(
+                cast(column, String).ilike(f"%{search_query}%", escape=escape_character)
+                for column in columns
+            )
+        )
+
+
 class BaseModel(object):
-    query: BaseQuery
+    query: SearchableQuery
```
Comment (gnudeb) [inline jetkit/db/__init__.py]: does this type annotation work for specifying a query class? I've only seen this style: `query_class = SearchableQuery`

Comment (revmischa) [reply jetkit/db/__init__.py]: it doesn't. i fixed it - https://flask-sqlalchemy.palletsprojects.com/en/2.x/customizing/#query-class

**Comment by revmischa** [comment]
https://pypi.org/project/JetKit-Flask/6.0.1/

**Pull request merged**